packages/slate-react/src/components/editable.tsx

```diff
@@ -613,7 +614,7 @@ export const Editable = (props: EditableProps) => {
               // aren't correct and never fire the "insertFromComposition"
               // type that we need. So instead, insert whenever a composition
               // ends since it will already have been committed to the DOM.
-              if (!IS_SAFARI && !IS_FIREFOX && event.data) {
+              if (!IS_SAFARI && !IS_FIREFOX && !IS_IOS && event.data) {
```

Comment:
This looks fine, but it's stale as we recently updated it to IS_FIREFOX_LEGACY. Could you do a quick rebase and then we can land this? Thanks @ulion!